src/wizard/apple/profiling-onboarding/ios/2.configure-performance.md

```diff
@@ -18,3 +18,13 @@ SentrySDK.start { options in
     options.tracesSampleRate = 1.0
 }
 ```
+
+```objc {tabTitle:Objective-C}
```

Comment:
I think the wizard does not support tab titles, at least it used to be like that before :) but I guess you cannot test it without deploying

Reply:
Thanks for pointing that out! Yep, I see it in the wizard. I'll just make the change for the docs.